src/crimson/os/seastore/onode.h

```diff
@@ -32,6 +32,7 @@ struct onode_layout_t {
   ceph_le32 oi_size{0};
   ceph_le32 ss_size{0};
   omap_root_le_t omap_root;
+  omap_root_le_t xattr_root;
```

Comment:
is onode_layout_t  a memory structure or disk content layout structure?

Reply:
disk content layout struction

Reply:
cc @athanatos 

Ack, we will have separate roots for omap and xattrs.

The advantage over single root is that we don't need special prefixes to distinguish between omap keys and xattr keys.

Reply:
Yeah, this what I'd expect.